### ISSUE TITLE
Details on removing the potentiometer and soldering the wire

### ISSUE THREAD

**Issue by Tomasonx** [electrical-pro/ESP32_Robot#3]
I have already tried on 3 pieces of DRV8825.

Can you detail the process and possibly check if the soldering was correct?

**Comment by electrical-pro** [comment]
@Tomasonx, Hello, what is the problem exactly?, just use a soldering iron, use flux and solder and remove the potentiometer.
then solder the wire like on the picture
![image](https://user-images.githubusercontent.com/31592485/166574593-1dae3c07-69d8-4728-b9d7-6d0365ac96dd.png)

everything according to the circuit
![image](https://user-images.githubusercontent.com/31592485/166574907-b28cfe25-c713-475c-8186-b71c9dd0f507.png)

**Comment by Tomasonx** [comment]
1. Removing the potentiometer = no problem.
2. After that you have 3 points, VCC, GND, Output.
From the picture I understand that you are soldering only with point 3 (Output), right?
Because it's a very small point, how do you test (measure) that you're soldering ok?

![2](https://user-images.githubusercontent.com/11645510/166632680-d06da73b-4920-465d-aa82-18e075f105b9.png)

**Comment by electrical-pro** [comment]
@Tomasonx, yes only point 3 (Output). You can use a magnifying glass to check visually.
You can use a multimeter to check if you don't have a short circuit between VCC, GND, and Output.
If you want to check more you can follow that PCB trace.
